fsck/fsck.c

```diff
@@ -670,58 +671,72 @@ static int read_boot_region(struct exfat_blk_dev *bd, struct pbr **pbr,
 		return -ENOMEM;
 	}
 
+	if (main_sect)
```

Comment:
main_sect is needed ? you can use bs_offset as argument like before

Reply:
Okay, I will change it.